scripts/sv/svDiscover.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -eu
+
+if [[ "$#" -lt 7 ]]; then
+    echo -e "Please provide:"
+    echo -e "  [1] local directory of GATK build (required)"
+    echo -e "  [2] cluster name (required)"
+    echo -e "  [3] output directory on the cluster (required)"
+    echo -e "  [4] absolute path to the BAM on the cluster (index assumed accompanying the bam) (required)"
+    echo -e "  [5] absolute path to the reference fasta on the cluster (required)"
+    echo -e "  [6] absolute path to the 2-bit reference on the cluster (required)"
```

Comment:
done, except for the index image fie because it lives on local file systems rather than hdfs, and except for the "....dups" file.